moduleroot/CONTRIBUTING.md.erb

```diff
@@ -1,3 +1,13 @@
+Source for this document
```

Comment:
This should be at the very bottom IMHO. It's the least relevant part of the document for someone who wants to contribute to the module itself.

Reply:
:+1: